suplemon/file.py

```diff
@@ -141,6 +141,7 @@ def _read_text(self, file):
             f.close()
             return data
         except:
+            self.logger.exception("Reading file failed.")
```

Comment:
I'm not sure if the exception will include a filepath, it might be good to inline one in the error message. For example:

```python
self.logger.exception("Failed reading file \"{file}\"".format(file=file))
```